Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/AddOfferFragment.java

```diff
@@ -286,19 +292,21 @@ public void onResponse(Call<CreateResponse> call, Response<CreateResponse> respo
                     realm.commitTransaction();
                     realm.close();
                 } else {
-                    Converter<ResponseBody, CreateError> converter = VolontuloApp.retrofit.responseBodyConverter(CreateError.class, new Annotation[0]);
+                    Converter<ResponseBody, SaveError> converter = VolontuloApp.retrofit.responseBodyConverter(SaveError.class, new Annotation[0]);
                     try {
-                        final CreateError error = converter.convert(response.errorBody());
+                        final SaveError error = converter.convert(response.errorBody());
                         Log.d(TAG, "ERROR: " + error.getDetail());
                     } catch (IOException e) {
                         Log.d(TAG, "EXCEPTION: " + e.getMessage());
```

Comment:
It is good to describe what kind of error occured via `message` parameter and pass `e` as third parameter to logger method.